lmdeploy/messages.py

```diff
@@ -96,7 +96,6 @@ class TurbomindEngineConfig:
     """TurboMind Engine config.
 
     Args:
-        model_name (str): the name of the deployed model
```

Comment:
may need to update docs like here:
https://lmdeploy.readthedocs.io/en/latest/inference/pipeline.html#turbomindengineconfig